tasks/ragel.rake

```diff
@@ -4,23 +4,23 @@ load 'lib/mail/parsers.rb'
 rule 'lib/mail/parsers/rfc5234_abnf_core_rules.rl' => 'lib/mail/parsers/rfc3629_utf8.rl'
 
 # All RFC 5322 parsers depend on ABNF core rules
-rule /rfc5322_.+\.rl\z/ => 'lib/mail/parsers/rfc5234_abnf_core_rules.rl'
+rule (/rfc5322_.+\.rl\z/) => 'lib/mail/parsers/rfc5234_abnf_core_rules.rl'
```

Comment:
I think that blank after the `rule` method appear unnecessary.

```diff
-rule /rfc5322_.+\.rl\z/ => 'lib/mail/parsers/rfc5234_abnf_core_rules.rl'
+rule(/rfc5322_.+\.rl\z/) => 'lib/mail/parsers/rfc5234_abnf_core_rules.rl'
```

Other similar changes are also the same.

Reply:
This blank is necessary.
If I deleted it, the results are as follows.

``` ruby
SyntaxError: /mail/tasks/ragel.rake:7: syntax error, unexpected =>, expecting end-of-input
rule(/rfc5322_.+\.rl\z/) => 'lib/mail/parsers/rfc5234_abn

/mail/Rakefile:23:in `load'
/mail/Rakefile:23:in `block in <top (required)>'
/mail/Rakefile:23:in `each'
/mail/Rakefile:23:in `<top (required)>'
/.rbenv/versions/2.4.1/bin/bundle:22:in `load'
/.rbenv/versions/2.4.1/bin/bundle:22:in `<main>'
```

Reply:
Oops. I was wrong. However, I think that the change enclosing `/.../` with `(...)` seems to be redundant. In my opinion, it might be better to use a regexp literal of `%r|...|`.

Reply:
I fix using percent literal.
Percent literals were used in other codes, so I thought it was appropriate.